RadarSDK/RadarAPIClient.h

```diff
@@ -86,6 +87,7 @@ typedef void (^_Nonnull RadarSyncLogsAPICompletionHandler)(RadarStatus status);
         attestationString:(NSString *_Nullable)attestationString
                     keyId:(NSString *_Nullable)keyId
          attestationError:(NSString *_Nullable)attestationError
+                encrypted:(BOOL)encrypted
```

Comment:
was going to suggest we think about changing this to `verified` given discussion in slack & recent changes, but of course that's already used.. 🤔

Reply:
Looking at the JWT docs, I think `encoded` would perhaps be most accurate

Reply:
`token` param or `POST /track/token` endpoint are both options, too